tensorflow/python/ops/sparse_ops.py

```diff
@@ -2733,6 +2734,109 @@ def sparse_transpose(sp_input, perm=None, name=None):
     return transposed_st
 
 
+@tf_export("sparse.map_values", v1=[])
+@dispatch.add_dispatch_support
+def map_values(op, *args, **kwargs):
+  """Applies `op` to the `.values` tensor of one or more `SparseTensor`s.
+
+  Replaces any `SparseTensor` in `args` or `kwargs` with its `values`
+  tensor (which contains the non-default values for the SparseTensor),
+  and then calls `op`.  Returns a `SparseTensor` that is constructed
+  from the input `SparseTensor`s' `indices`, `dense_shape`, and the
+  value returned by the `op`.
+
+  If the input arguments contain multiple `SparseTensor`s, then they must have
+  equal `indices` and dense shapes.
+
+  Examples:
+
+  >>> s = tf.sparse.from_dense([[1, 2, 0],
+  ...                           [0, 4, 0],
+  ...                           [1, 0, 0]])
+  >>> print(tf.sparse.to_dense(tf.sparse.map_values(tf.ones_like, s)).numpy())
+  [[1 1 0]
+   [0 1 0]
+   [1 0 0]]
+  >>> print(tf.sparse.to_dense(tf.sparse.map_values(tf.multiply, s, s)).numpy())
+  [[ 1  4  0]
+   [ 0 16  0]
+   [ 1  0  0]]
+  >>> print(tf.sparse.to_dense(tf.sparse.map_values(tf.add, s, 5)).numpy())
+  [[6 7 0]
+   [0 9 0]
+   [6 0 0]]
```

Comment:
You don't need the `print`. This is like a normal Python REPL, an expression that evaluates to a value results in the value getting printed.

Probably not needing `numpy()` either, as that does not exist if the tensor is built in graph mode. But let's go with it.